### ISSUE TITLE
`jQuery.ajax.restore();` doesn't work

### ISSUE THREAD

**Issue by 76784** [sinonjs/sinon#2001]
[On this documentaion page](https://sinonjs.org/#get-started) (see "Testing Ajax"), the following example is provided:

> The following function triggers network activity:

```
function getTodos(listId, callback) {
    jQuery.ajax({
        url: '/todo/' + listId + '/items',
        success: function (data) {
            // Node-style CPS: callback(err, data)
            callback(null, data);
        }
    });
}
```
> To test this function without triggering network activity we could replace `jQuery.ajax`
```
after(function () {
    // When the test either fails or passes, restore the original
    // jQuery ajax function (Sinon.JS also provides tools to help
    // test frameworks automate clean-up like this)
    jQuery.ajax.restore();
});
it('makes a GET request for todo items', function () {
    sinon.replace(jQuery, 'ajax', sinon.fake());
    getTodos(42, sinon.fake());

    assert(jQuery.ajax.calledWithMatch({ url: '/todo/42/items' }));
});
```
**When testing this using the following setup, `jQuery.ajax.restore();` returns "TypeError: jQuery.ajax.restore is not a function". Here is the setup:**
`package.json`
```
  "devDependencies": {
    "browserify": "^16.2.3",
    "jquery": "^3.3.1",
    "mocha": "^6.0.2",
    "sinon": "^7.3.0"
  }
```
`testrunner.html`
```
<body>
	<div id="mocha"></div>
	<script src="node_modules/mocha/mocha.js"></script>
	<script src="node_modules/sinon/pkg/sinon.js"></script>
	<script>mocha.setup('bdd')</script>
	<script src="bundle.js"></script>
	<script>
		mocha.run();
	</script>
</body>
```
`test.js`
```
var assert = require('assert'); 
var jQuery = require('jquery');

function getTodos(listId, callback) {
  jQuery.ajax({
      url: '/todo/' + listId + '/items',
      success: function (data) {
          // Node-style CPS: callback(err, data)
          callback(null, data);
      }
  });
}
describe('getTodos()', function() {
  //https://sinonjs.org/#get-started
  //Testing Ajax
  after(function () {
      jQuery.ajax.restore(); // returns "TypeError: jQuery.ajax.restore is not a function"
  });
  it('makes a GET request for todo items', function () {
      sinon.replace(jQuery, 'ajax', sinon.fake());
      getTodos(42, sinon.fake());

      assert(jQuery.ajax.calledWithMatch({ url: '/todo/42/items' }));
  });
});
```
run: `browserify test.js --outfile bundle.js`

**Comment by 76784** [comment]
I just saw on another thread that `sinon.restore()` is deprecated. I'll edit the question to remove that.

**Comment by 76784** [comment]
> I just saw on another thread that `sinon.restore()` is deprecated. I'll edit the question to remove that.

So actually, I'm not sure where I saw the thread that said this, and I may be wrong, because I see `sinon.restore()` in the documentation. Whatever the case, I'll leave `sinon.restore()` to the side for the moment.

**Comment by mantoni** [comment]
To clarify: `sinon.restore()` used to have a different meaning, was deprecated and then removed. Since v5, the `sinon` object is a default sandbox instance, which comes with a `restore` function. Confusing history, I know.

**Comment by fatso83** [comment]
Hi, this is a bug in the documentation. It has happened when someone updated the examples from using `sinon.stub()` to using the new fakes concept. Fakes do not have  a `restore` method, so you need to restore using the sandbox, which means instead of `ajax.restore()`, it should be `sinon.restore()`.

Would you like to do the update of the docs?

**Comment by 76784** [comment]
Sure, I can update it. I just tried pushing but get a 403 error. Is there a way I can get permission? Do you need a feature branch for your pull requests?

**Comment by mroderick** [comment]
> Sure, I can update it. I just tried pushing but get a 403 error. Is there a way I can get permission? Do you need a feature branch for your pull requests?

You'll need to fork the repository, and make the pull request from your fork

**Comment by fatso83** [comment]
https://help.github.com/en/articles/creating-a-pull-request

**Comment by mantoni** [comment]
Addressed by #2004.
